tests/system/feature_store/test_feature_store.py

```diff
@@ -831,6 +832,84 @@ def test_unaggregated_columns(self):
         }
     )
 
+    def test_schedule_on_filtered_by_time(self):
```

Comment:
some more tests permutations - 
1. add data in the past (before last_written) and see it's not read
2. use several targets in different order
3. verify partitioned parquet